ports/espressif/common-hal/esp32_camera/Camera.c

```diff
@@ -57,6 +61,11 @@ void common_hal_esp32_camera_camera_construct(
     mp_int_t framebuffer_count,
     camera_grab_mode_t grab_mode) {
 
+    if (common_hal_espidf_get_reserved_psram() == 0) {
+        mp_raise_msg(&mp_type_MemoryError, translate(
+            "esp32_camera.Camera requires reserved PSRAM to be configured. "
+            "See the documentation for instructions.");
```

Comment:
```suggestion
            "See the documentation for instructions."));
```
Missing a `)`

Reply:
there's not an "egg on face" emoji but there should be